src/string/i18n.c

```diff
@@ -12,11 +12,11 @@ char *cr_translate_assert_msg(int msg_index, ...) {
     static char *messages[] = {
         [CRITERION_ASSERT_MSG_FAIL] = N_("The conditions for this assertion were not met."),
         [CRITERION_ASSERT_MSG_EXPR_FALSE] = N_("The expression %s is false."),
-        [CRITERION_ASSERT_MSG_EXPR_AS_STRINGS_FALSE] = N_("The expression (as strings) %s is false."),
+        [CRITERION_ASSERT_MSG_EXPR_AS_STRINGS_FALSE] = N_("The expression (as strings) %s is false: actual=`%s` expected=`%s`."),
```

Comment:
I18n strings with multiple printf specifiers must provide positional specifiers. See https://github.com/kaidowei/Criterion/blob/51dcb9a16dab076b748d9b9bb8f2c380a6189caa/src/string/i18n.c#L23-L33 for examples.